packages/page-accounts/src/modals/Create.tsx

```diff
@@ -394,7 +394,8 @@ function Create ({ className = '', onClose, onStatusChange, seed: propsSeed, typ
                 <Modal.Columns>
                   <Modal.Column>
                     <Input
-                      help={t<string>('You can set a custom derivation path for this account using the following syntax "/<soft-key>//<hard-key>". The "/<soft-key>" and "//<hard-key>" may be repeated and mixed`. An optional "///<password>" can be used with a mnemonic seed, and may only be specified once.')}
+                      help={t<string>(pairType === 'ethereum' ? 'You can set a custom derivation path for this account using the following syntax "m/<purpose>/<coin_type>/<account>/<change>/<address_index>' : 'You can set a custom derivation path for this account using the following syntax "/<soft-key>//<hard-key>". The "/<soft-key>" and "//<hard-key>" may be repeated and mixed`. An optional "///<password>" can be used with a mnemonic seed, and may only be specified once.')}
```

Comment:
We cannot put the conditional inside the `t('...`)` so needs to take the following form -

```
<Input
  help={
    pairType === 'ethereum'
      ? t<string>('...')
      : t<string>('....')
  }
  ...
```